docs/operations/github_ci_app.rst

```diff
@@ -0,0 +1,73 @@
+######################################
+Adding a new GitHub CI app integration
+######################################
+
+
+Create a new GitHub app
+=======================
+
+#. Click the ``New GitHub App`` button in the `lsst-sqre org Developer Settings apps page <https://github.com/organizations/lsst-sqre/settings/apps>`__.
+#. Name it ``mobu CI (<env URL or id if the URL is too long>)``.
+#. Make sure the ``Active`` checkbox is checked in the ``Webhook`` section.
+#. Enter ``https://<env URL>/mobu/github/ci/webhook`` in the ``Webhook URL`` input.
+#. Generate a strong password to use as the webhook secret.
+#. Store this in the ``SQuaRE`` vault in the ``LSST IT`` 1Password account in an item named ``mobu GitHub CI app webhook secret (<env URL>)``.
+#. Get this into the Phalanx secret store for that env under the key: ``github-ci-app-webhook-secret`` (`this process <https://phalanx.lsst.io/admin/add-new-secret.html>`__ is different for different envs).
+#. Enter this secret in the ``Webhook secret (optional)`` box in the GitHub App config.
+#. Select ``Read and Write`` in the dropdown of the ``Checks`` access category in the ``Repository Permissions`` section.
+#. Select ``Read-only`` in the dropdown of the ``Contents`` access category in the ``Repository Permissions`` section.
+#. Check the ``Check suite`` and ``Check run`` checkboxes in the ``Subscribe to events`` section.
+#. Select the ``Any account`` radio button in the ``Where can this GitHub App be installed?`` section.
+#. Click the ``Create GitHub App`` button.
+#. Find the ``App ID`` (an integer) in the ``About`` section. Get this into the Phalanx secret store for that env under the key: ``github-ci-app-id`` (`this process <https://phalanx.lsst.io/admin/add-new-secret.html>`__ is different for different envs).
+#. Click the ``Generate a private key`` button in the ``Private keys`` section.
+#. Store this private key in the ``SQuaRE`` vault in the ``LSST IT`` 1Password account in an item named ``mobu GitHub CI app private key (<env URL>)``.
+#. Get this into the Phalanx secret store for that env under the key: ``github-ci-app-private-key`` (`this process <https://phalanx.lsst.io/admin/add-new-secret.html>`__ is different for different envs).
+#. Do the `Phalanx configuration <#phalanx-configuration>`__.
```

Comment:
With Sphinx, generally you would put:
```
.. _phalanx-configuration:
```
on the line before the heading and then use:
```suggestion
#. Do the :ref:`Phalanx configuration <phalanx-configuration>`.
```
You can omit the `Phalanx configuration` part and the angle brackets if the anchor text is the same as the heading text. This allows Sphinx to do better validation of the internal links at build time, and also keeps working even if the section is moved to a different file.